test/controllers/user.test.js

```diff
@@ -0,0 +1,28 @@
+const supertest = require('supertest');
+const { factory } = require('factory-girl');
+
+const app = require('../../app');
+
+const request = supertest(app);
+
+require('../factory/user');
+
+describe('Users Controller', () => {
+  describe('/POST users', () => {
+    describe('when using valid parameters', () => {
```

Comment:
You have to add more tests cases (invalid password, duplicated user, invalid email, etc)